composer.json

```diff
@@ -0,0 +1,50 @@
+{
+	"name": "robincornett/genesis-accessible",
+	"type": "wordpress-plugin",
+	"description": "Test plugin",
+	"keywords": [
+		"wordpress",
+		"genesis",
+		"accessibility"
+	],
+	"homepage": "https://github.com/robincornett/genesis-accessible",
+	"license": "GPL-2.0-or-later",
+	"authors": [
+		{
+			"name": "Rian Rietveld",
+			"homepage": "https://rianrietveld.com/",
+			"role": "Developer"
+		},
+		{
+			"name": "Robin Cornett",
+			"homepage": "https://robincornett.com/",
+			"role": "Developer"
+		}
+	],
+	"require": {
+		"php": "^5.6 || ^7",
+		"composer/installers": "^1"
+	},
+	"require-dev": {
+		"dealerdirect/phpcodesniffer-composer-installer": "^0.4.4",
+		"phpcompatibility/phpcompatibility-wp": "^1",
+		"roave/security-advisories": "dev-master",
+		"squizlabs/php_codesniffer": "^3.3",
+		"wp-coding-standards/wpcs": "^1"
+	},
+	"config": {
+		"sort-packages": true
+	},
+	"minimum-stability": "dev",
+	"prefer-stable": true,
+	"scripts": {
+		"install-codestandards": [
+			"Dealerdirect\\Composer\\Plugin\\Installers\\PHPCodeSniffer\\Plugin::run"
+		],
+		"phpcs": "./vendor/bin/phpcs && ./vendor/bin/phpcs ./genesis-accessible.php -sp --standard=PHPCompatibilityWP --runtime-set testVersion 5.2"
```

Comment:
After checking out this branch (or merging), you can run `composer install`. This will install all of the `require-dev` packages.

You can then run `./vendor/bin/phpcs` and get the basic PHPCS results.

Although the plugin requires PHP 5.6, it's good to ensure that the main plugin file is PHP 5.2 compatible (it currently is), and then do a PHP version check before including another file which includes the rest of the PHP 5.6 syntax code (doesn't do that yet).

As such, instead of running PHPCS directly, you can run `composer phpcs`, and that will run PHPCS twice; once as per the `.phpcs.xml.dist` config, and a second time just on the main plugin file checking for PHP 5.2 compatibility.

Reply:
Thank you for putting this together, Gary, and for explaining things! I'm having a couple hiccups running things as easily as I think I ought to be able to--I think I'm having a bit of conflict with my global phpcs install, on one point, and then experiencing an issue with one of the sniffs when it runs. But I'm assuming the issues are on my end, not with your PR.

Reply:
Let's jump on a call today and work through it.

Reply:
Thanks, Gary--I should be available after 11:30EST, which may be pushing it for you, but if not, yes please.